docs/client_guide/client_guide.md

```diff
@@ -60,6 +60,8 @@ The important configuration of client is listed as following. These configuratio
 | <client_type>.rss.client.rpc.netty.maxOrder                     | 3                                      | The value of maxOrder for PooledByteBufAllocator when using gRPC internal Netty on the client-side. This configuration will only take effect when rss.rpc.server.type is set to GRPC_NETTY.                                                                                                                                                                                                                                                                                                  |
 | <client_type>.rss.client.rpc.netty.smallCacheSize               | 1024                                   | The value of smallCacheSize for PooledByteBufAllocator when using gRPC internal Netty on the client-side. This configuration will only take effect when rss.rpc.server.type is set to GRPC_NETTY.                                                                                                                                                                                                                                                                                            |
 | <client_type>.rss.client.blockIdManagerClass                    | -                                      | The block id manager class of server for this application, the implementation of this interface to manage the shuffle block ids                                                                                                                                                                                                                                                                                                                                                              |
+| <client_type>.rss.client.reportExcludeProperties                | -                                      | the report exclude properties could be configured by this option                                                                                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
Is the document right about `reportExcludeProperties` and `reportIncludeProperties`?